numba/blackscholes/GPU/bs_erf_numba_kernel.py

```diff
@@ -2,15 +2,31 @@
 #
 # SPDX-License-Identifier: MIT
 
+import os
 import dpctl
 import base_bs_erf
-import numba_dppy
+from device_selector import get_device_selector
 from math import log, sqrt, exp, erf
 
-# blackscholes implemented using dppy.kernel
-@numba_dppy.kernel(
-    access_types={"read_only": ["price", "strike", "t"], "write_only": ["call", "put"]}
-)
+backend = os.getenv("NUMBA_BACKEND", "legacy")
+if backend == "legacy":
+    import numba_dppy
+    from numba_dppy import kernel, get_global_id, DEFAULT_LOCAL_SIZE
+
+    __kernel = numba_dppy.kernel(
```

Comment:
BTW, latest dpcomp kernel decorator now supports `fastmath` and `access_types` (which is ignored) args.